src/main/java/FormController.java

```diff
@@ -8,21 +7,27 @@
 
 public class FormController implements Controller {
     private String resourcePath;
+    private String[] methodsAllowed;
+    private final String NEWLINE = "\r" + System.lineSeparator();
+    private final String REQUEST_SUCCEEDED = "HTTP/1.1 200 OK";
```

Comment:
Since you'll probably be using this in other controllers that will return a 200 OK, it might make sense to pull this out into its own class.